deployments/docker/Dockerfile

```diff
@@ -1,22 +1,23 @@
-FROM gitlab-registry.cern.ch/linuxsupport/alma9-base:20240201-2
+FROM gitlab-registry.cern.ch/linuxsupport/alma9-base:20240501-1
 
 # Setup EOS repository.
-ADD deployments/docker/eos9al-stable.repo /etc/yum.repos.d/eos9al-stable.repo
-RUN yum install -y dnf-plugins-core epel-release \
-    && dnf config-manager --set-enabled crb
+COPY deployments/docker/eos9al-stable.repo /etc/yum.repos.d/eos9al-stable.repo
 
-# Install the EOS client and autofs-related packages.
-RUN yum install -y --setopt=skip_missing_names_on_install=False \
-        eos-fusex-5.2.22                                        \
-        autofs                                                  \
-        cern-eos-autofs                                         \
-        cern-eos-autofs-squashfs                                \
-	&& yum clean all
-
-# eosxd-csi deployment with Helm should provide
-# its own autofs configuration. The files listed below
-# break certain mounts, and so need to be removed.
-RUN rm /etc/auto.eos.main.misc    \
+# Install the EOS client, autofs-related packages and remove autofs
+# configuration files, as they break certain mounts.
+#
+# eosxd-csi helm deployment comes packaged with it's own autofs configuration.
+RUN yum install -y \
+        dnf-plugins-core \
+        epel-release && \
+    dnf config-manager --set-enabled crb && \
+    yum install -y --setopt=skip_missing_names_on_install=False \
+        autofs \
+        cern-eos-autofs \
+        cern-eos-autofs-squashfs \
+        eos-fusex-5.2.22 && \
+	yum clean all && \
```

Comment:
use dnf instead of yum

What do you think about removing /var/cache/dnf /usr/share/man /usr/local/share/man? they are ~90Kb each

Reply:
sounds good - have swapped over to dnf